src/scenes/home/getInvolved/check.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 105.5 110.4" style="enable-background:new 0 0 105.5 110.4;" xml:space="preserve">
+<g transform="translate(-349.14 -202.1)">
+	<path d="M400.9,208.2c-28.5,0-51.8,23.2-51.8,51.8s23.2,51.8,51.8,51.8s51.8-23.2,51.8-51.8c0-12.3-4.2-23.4-11.3-32.4l-4.9,5.8
+		c6.2,7.6,8.6,16.1,8.6,26.6c0,24.5-19.8,44.2-44.2,44.2s-44.2-19.8-44.2-44.2s19.8-44.2,44.2-44.2c7.7,0,13.5,1.1,19.9,4.6l4.6-6.1
+		C418,210.2,410,208.2,400.9,208.2L400.9,208.2z"/>
+	<path d="M381.8,253.2c2.1,0,3.7,1.7,4.7,5.1c2.1,6.4,3.7,9.6,4.6,9.6c0.7,0,1.4-0.5,2.2-1.6c15-24.1,29-43.6,41.8-58.5
+		c3.3-3.9,8.6-5.8,15.8-5.8c1.7,0,2.9,0.2,3.5,0.5c0.6,0.3,0.9,0.7,0.9,1.2c0,0.7-0.9,2.2-2.6,4.4c-20.7,24.8-39.8,51.1-57.5,78.7
+		c-1.2,1.9-3.7,2.9-7.5,2.9c-3.9,0-6.1-0.2-6.8-0.5c-1.8-0.8-4-4.9-6.4-12.3c-2.8-8.2-4.2-13.3-4.2-15.4c0-2.2,1.9-4.4,5.6-6.5
+		C377.9,253.8,380,253.2,381.8,253.2"/>
+</g>
+</svg>
```

Comment:
Are this file and 'src/images/icons/check.svg' both needed? I only see one reference to a check.svg added in this PR.